test/patterns/prime generator/prime.perfcompopts

```diff
@@ -0,0 +1 @@
+-sexecTime=true
```

Comment:
This and the next file should end with a newline.

Also, while this will generate performance timings for us nightly, it doesn't include any way for us to track what that looks like visually.  For that, you'll need a .graph file (and to add that .graph file to test/GRAPHFILES).  The details on how to do that should also be in the TestSystem document.